doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -710,6 +711,7 @@ Matter samples
     See the :ref:`migration guide <migration_3.2_required>` for more information.
   * All Matter samples that support low-power mode to use the :ref:`lib_ram_pwrdn` feature with the nRF54LM20 DK.
     This change resulted in decreasing the sleep current consumption by more than two uA.
+  * All Matter samples to use the code-driven approach for the Identify cluster implementation instead of the zap-driven approach.
```

Comment:
I'm wondering if we could use this commit, as an example for the migration guide?